messaging-activemq/subsystem/src/test/resources/org/wildfly/extension/messaging/activemq/subsystem_17_0_reject_transform.xml

```diff
@@ -0,0 +1,287 @@
+<subsystem xmlns="urn:jboss:domain:messaging-activemq:16.0">
```

Comment:
This configuration should include the `commit-interval` attribute, and the test should assert that those resources were rejected; otherwise, we will be only testing the default values, which I understand are already tested using subsystem_17_0_transform.xml